includes/container-service-faq.md

```diff
@@ -92,6 +92,16 @@ ssh userName@masterFQDN –A –p 22 
 
 For more information, see [Connect to an Azure Container Service cluster](../articles/container-service/kubernetes/container-service-connect.md).
 
+### My DNS name resolution isn't working. What should I do?
```

Comment:
Perhaps add to the title the word `Windows` to give some indication that this is specific to the platform.

Reply:
Great point, will do